src/main/java/org/jabref/logic/journals/JournalAbbreviationRepository.java

```diff
@@ -125,4 +126,9 @@ public void addCustomAbbreviations(Collection<Abbreviation> abbreviationsToAdd)
     public Set<String> getFullNames() {
         return fullToAbbreviation.keySet();
     }
+
+    public List<Abbreviation> getBuiltin() {
```

Comment:
I think this is a wrong semantics here. The built-in refers to the built-in journal list, but `fullToAbbreviation` is read from the file passed as the constructor argument.